src/components/formio/radio.stories.ts

```diff
@@ -88,3 +88,36 @@ export const WithErrors: Story = {
     await expect(canvas.queryByText('Example error')).toBeInTheDocument();
   },
 };
+
+export const DefaultValueRemoved: Story = {
+  args: {
```

Comment:
I don't see the `isClearable` prop being passed that should enable this behaviour?